dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -957,6 +957,10 @@ ERROR_NO_DERIVED_PLATFORM=CWWKF1516E: The platform couldn't be determined. The f
 ERROR_NO_DERIVED_PLATFORM.explanation= The platform was not set or couldn't be derived from existing versioned features. 
 ERROR_NO_DERIVED_PLATFORM.useraction=Add a platform for versionless features.
 
+ERROR_VERSIONLESS_INSTALL=CWWKF1517E: The installFeature command does not support versionless features.
+ERROR_VERSIONLESS_INSTALL.explanation= The installFeature command does not support versionless features.
+ERROR_VERSIONLESS_INSTALL.useraction=Use installServerFeatures command.
```

Comment:
Suggested rewrite:
Use the installServerFeatures command, which does not require features to be versioned, or install a different feature that is not versionless.